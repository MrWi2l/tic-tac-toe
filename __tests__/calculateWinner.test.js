const calculateWinner = require('../src/calculateWinner')

test('DRAW!', () => {
  expect(calculateWinner([
    'X', 'O', 'X',
    'O', 'O', 'X',
    'O', 'X', 'O',
  ])).toMatchSnapshot()
})

test('X is Winner', () => {
  expect(calculateWinner([
    'X', 'X', 'X',
    '', '', '',
    '', '', '',
  ])).toMatchSnapshot()
})

test('X is Winner', () => {
  expect(calculateWinner([
    'X', '', '',
    'X', '', '',
    'X', '', '',
  ])).toMatchSnapshot()
})

test('X is Winner', () => {
  expect(calculateWinner([
    '', '', 'X',
    '', '', 'X',
    '', '', 'X',
  ])).toMatchSnapshot()
})

test('X is Winner', () => {
  expect(calculateWinner([
    '', '', '',
    '', '', '',
    'X', 'X', 'X',
  ])).toMatchSnapshot()
})

test('X is Winner', () => {
  expect(calculateWinner([
    'X', '', '',
    '', 'X', '',
    '', '', 'X',
  ])).toMatchSnapshot()
})

test('X is Winner', () => {
  expect(calculateWinner([
    '', 'X', '',
    '', 'X', '',
    '', 'X', '',
  ])).toMatchSnapshot()
})

test('X is Winner', () => {
  expect(calculateWinner([
    '', '', 'X',
    '', 'X', '',
    'X', '', '',
  ])).toMatchSnapshot()
})

test('X is Winner', () => {
  expect(calculateWinner([
    '', '', '',
    'X', 'X', 'X',
    '', '', '',
  ])).toMatchSnapshot()
})
