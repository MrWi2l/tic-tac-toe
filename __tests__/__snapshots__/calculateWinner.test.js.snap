exports[`test DRAW! 1`] = `null`;

exports[`test X is Winner 1`] = `"X"`;

exports[`test X is Winner 2`] = `"X"`;

exports[`test X is Winner 3`] = `"X"`;

exports[`test X is Winner 4`] = `"X"`;

exports[`test X is Winner 5`] = `"X"`;

exports[`test X is Winner 6`] = `"X"`;

exports[`test X is Winner 7`] = `"X"`;

exports[`test X is Winner 8`] = `"X"`;
